### PR TITLE
Feat/display answers

### PR DESCRIPTION
**Pull request by makkoman**
WIP-215 - Tabs layout for the open questions and the list of the answered ones.
#### What does this PR do?
Integrates a tab component and a list of answered questions
#### Where should the reviewer start?
Install the new scripts included in the package.json file
#### How should this be manually tested?
once the new packages are installed, just run the project and enter to [http://localhost:3000/all_questions](http://localhost:3000/all_questions) there should appear two tabs. Clicking in each tab should change between the table of questions and the answered ones
#### Any background context you want to provide?
QuestionsAnswered.js was builded with List.js as base.
#### What are the relevant tickets?
[https://wizeline.atlassian.net/browse/WIP-215](WIP-215)
#### Screenshots
screencapture-localhost-3000-all_questions-1495143135656
screencapture-localhost-3000-all_questions-1495143158739

### CODE REVIEW

**Comment by alan-maldonado** [comment]
LGTM just, resolve the conflicts files

File: src/components/List.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ class List extends Component {
   }
 
   componentDidMount() {
-    axios.get('http://51d675a7.ngrok.io/list')
+    axios.get('https://api.myjson.com/bins/mlh29')
```
Comment (rominavarela) [inline src/components/List.js]: You are using this value in many places. Please make a reusable CONST for the host. Use this as example https://github.com/wizeline/wize-q-front/blob/master/src/components/QuestionInput.js#L5

File: src/components/QuestionsAnswered.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
+/* eslint react/jsx-no-bind: 0 */  // --> OFF
```
Comment (rominavarela) [inline src/components/QuestionsAnswered.js]: Please avoid this directives, I see this is not used in other components, so the rule shouldn't be a problem.

File: src/components/QuestionsAnswered.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+import React, { Component } from 'react';
+import { Table, Col } from 'react-bootstrap';
+import axios from 'axios';
+import '../styles/List.css';
+import '../styles/QuestionsAnswered.css';
+
+class QuestionsAnswered extends Component {
+  static renderAnswerRow(question) {
```
Comment (rominavarela) [inline src/components/QuestionsAnswered.js]: Don't use static, by convention, we are using arrow functions. You can use this as example https://github.com/wizeline/wize-q-front/blob/master/src/components/QuestionInput.js#L21

File: src/components/List.js (hunk 1 above)
Comment (makkoman) [inline src/components/List.js]: we are using myjson.com to mock some json responses, it is used in two places, but they are different URLS, one is https://api.myjson.com/bins/mlh29 and the other one is https://api.myjson.com/bins/135a7p
Both of them will change once we connect the front end with the back end

**Comment by alan-maldonado** [comment]
Cool, 👍

File: src/components/List.js [hunk 4]
```diff
@@ -103,7 +100,7 @@ class List extends Component {
       </tr>,
     );
     return listItems;
-  }
+  };
```
Comment (codinronan) [inline src/components/List.js]: None of these semi colons are necessary - actually they suggest a convention we are not following, namely that they are assigning keys on an object `{}`, which they aren't (well - yes - but no).

Let's go ahead and remove them

Comment (makkoman) [inline src/components/List.js]: done! :D

**Pull request merged**